akka-stream/src/main/scala/akka/stream/impl/fusing/GraphStages.scala

```diff
@@ -37,8 +37,8 @@ object GraphStages {
    * INERNAL API
    */
   private[stream] abstract class SimpleLinearGraphStage[T] extends GraphStage[FlowShape[T, T]] {
-    val in = Inlet[T]("in")
-    val out = Outlet[T]("out")
+    val in = Inlet[T](toString + ".in")
+    val out = Outlet[T](toString + ".out")
```

Comment:
Yes, it does (`class TakeWithin[T](timeout: FiniteDuration) extends SimpleLinearGraphStage[T]`).

```
wiring MaybeSource.out (2145a4f4) -> IdleInject.in (0435d8d4)
wiring IdleInject.out (5d67b4c9) -> TakeWithin.in (7ed139c8)
wiring TakeWithin.out (2ec5b7f7) -> headOption.in (562fb04d)
```